tests/cases/elements/image.slint

```diff
@@ -33,15 +33,26 @@ const TestCase &instance = *handle;
 assert_eq(instance.get_img_width(), 320.);
 assert_eq(instance.get_img_height(), 480.);
 assert(instance.get_test());
+
+auto img_search = slint::testing::ElementHandle::find_by_element_id(handle, "TestCase::img");
+assert_eq(img_search.size(), 1);
+auto img = img_search[0];
+assert_eq(*img.accessible_role(), slint::testing::AccessibleRole::Image);
```

Comment:
```suggestion
assert(*img.accessible_role() == slint::testing::AccessibleRole::Image);
```

the error is that slint::testing::AccessibleRole doesn't implement the stream operator (equivalent to Debug trait in rust)